include/cling/Interpreter/InvocationOptions.h

```diff
@@ -14,16 +14,30 @@
 #include <vector>
 
 namespace cling {
+
+  class CompilerOpts {
```

Comment:
InvocationOptions parses a lot of stuff one variant of CIFactory::createCI has no use for.